### DISCUSSION TITLE
Grafana Dashboard Available?

### DISCUSSION THREAD

**Discussion by secondmanveran** [soketi/pws#772] [Q&A]
@rennokki or anyone 

Is there a Grafana Dashboard available? I have Prometheus and Grafana installed and set up to scrape and display metrics via the soketi exporter ... but I'm not sure what metrics I'm supposed to be using to monitor application usage etc.

Any help would be appreciated.

**Comment by nathanatimu** [comment]
would indeed be handy to have
